src/App.js

```diff
@@ -0,0 +1,122 @@
+                                Bill total
+                            </label>
+                            <input
+                                id="total"
+                                className="input-reset ba b--black-20 pa2 mb2 db w-100"
+                                type="number"
+                                aria-describedby="total-desc"
+                                name="total"
+                                {...total}
+                            />
+                            <small
+                                id="total-desc"
+                                className="f6 black-60 db mb2"
+                            >
+                                Enter the bill total
+                            </small>
+                        </div>
+                    </div>
+
+                    <div className="fl w-100 w-100-m w-50-l pv1 pr0-m pr2-l">
+                        <div className="bg-white pv4">
+                            <label htmlFor="name" className="f6 b db mb2">
+                                Tip percentage
+                            </label>
+                            <input
+                                id="tip"
+                                className="input-reset ba b--black-20 pa2 mb2 db w-100"
+                                type="number"
+                                aria-describedby="tip-desc"
+                                name="tip"
+                                {...percentage}
+                            />
+                            <small id="tip-desc" className="f6 black-60 db mb2">
+                                Enter tip as percentage of the bill (e.g.
+                                entering 20 means "I want to add a tip of 20%")
+                            </small>
+                        </div>
+                    </div>
+                    <div className="pv1">
+                        <Button
+                            className="f6 link dim br1 ph3 pv2 mb2 dib white bg-black"
+                            onClick={submit}
+                        >
+                            Calculate!
+                        </Button>
+                    </div>
+                    {tip === 0 && (
+                        <div className="pv1">
+                            <h2 className="orange">
+                                Please enter some numbers above!
+                            </h2>
+                        </div>
+                    )}
+                    {tip !== 0 && tip !== null && !updating && (
+                        <div className="pv1">
+                            <h2 className="result">So...</h2>
+                            <h3>
+                                <span style={inlineStyle}>Total:</span> $
+                                {total.value} + <br />
+                                <span style={inlineStyle}>Tip:</span> $
+                                {Math.round(tip * 100) / 100}
```

Comment:
Rounding can be troublesome, especially when totals include decimals. You might consider using `.toFixed(2)` instead to allow decimals.